sql/conseil.sql

```diff
@@ -139,6 +139,17 @@ CREATE TABLE tezos.token_balances (
     PRIMARY KEY (token_id, address, block_level)
 );
 
+CREATE TABLE tezos.tezos_names (
+    name character varying PRIMARY KEY,
+    owner character varying,
+    resolver character varying,
+    registered_at timestamp,
+    registration_period integer,
+    modified boolean
+);
+
+CREATE INDEX tezos_names_resolver_idx ON tezos.tezos_names USING btree (resolver);
+CREATE INDEX tezos_names_owner_idx ON tezos.tezos_names USING btree (owner);
```

Comment:
We probably need an index on `name` as well since we'll need to get frequently get records by name.

Reply:
The `name` is primary key right now, isn't that indexed by default?